src/configuration.ts

```diff
@@ -6,15 +6,16 @@ import { Config } from '@/configuration.model';
 import { validateSync } from 'class-validator';
 import { plainToClass } from 'class-transformer';
 
-const getConfigFilePath = () => {
+const getConfigFilePath = (): string => {
+    const baseDir = join(process.env.NODE_DIRNAME || __dirname, 'config');
     switch (process.env.NODE_ENV) {
         case 'dev':
         case 'test':
-            return join(__dirname, 'config', 'dev.config.yaml');
+            return join(baseDir, 'dev.config.yaml');
```

Comment:
I don’t believe it's necessary to make changes to this file.

Reply:
please revert changes here

Reply:
@notTanveer,  This is currently not possible, because the root of the `e2e` is quite different from others. If you know a good trick to rectify this, it would be splendidly appreciated